setup.py

```diff
@@ -85,6 +85,7 @@ def run_tests(self):
         'twisted': ['plugins/teamcity_plugin.py'],
     },
 
+    install_requires=['flake8>=3'],
```

Comment:
Other clients of teamcity-messages (nose, pytest etc) should not install flake8, I think